elastic_appsearch.module

```diff
@@ -54,3 +57,27 @@ function elastic_appsearch_theme() {
     ]
     ];
 }
+
+/**
+ * Implements hook_node_update().
+ *
+ */
+function elastic_appsearch_node_insert($node) {
```

Comment:
Shouldn't this be `hook_node_insert` in docblock?

Reply:
Also have you consider `hook_entity_insert` and `hook_entity_update` and `hook_entity_delete` instead of node specific hooks?